lib/versioning/maven/index.spec.ts

```diff
@@ -304,6 +305,7 @@ describe('versioning/maven/index', () => {
     expect(isStable('Hoxton.RELEASE')).toBe(true);
     expect(isStable('Hoxton.SR')).toBe(true);
     expect(isStable('Hoxton.SR1')).toBe(true);
+    expect(isStable('1.4.12-java7')).toBe(false);
```

Comment:
Wouldn't it rollback existing upgrades which rely on this qualifier? Formally it's stable, can't we instead provide regex-based filtering for cases when `java7` is undesirable?